src/PlayersManager.ts

```diff
@@ -712,17 +712,16 @@ export class PlayersManager extends Dispatcher {
         const captionsTracks = tracks.find(
           (track) => track._kind === "subtitles"
         );
-        if (
-          captionsTracks ||
-          (audioTracks &&
-            audioTracks._language &&
-            audioTracks._language != undefined)
-        ) {
-          // this video has captions or audio
+        if (audioTracks && audioTracks._language && audioTracks._language != undefined) {
           const view = player.getView().parentElement.parentElement
             .parentElement;
           view.classList.add("has-extra-tracks");
         }
+        if (captionsTracks){
+          const view = player.getView().parentElement.parentElement
```

Comment:
i don't like the .parentElement thing. this will break if the structure ever changes. why not do getElementById for targetId ?

Reply:
I don't want to change old functionality, this is the way it worked before so i went with the same logic.